manifests/init.pp

```diff
@@ -349,7 +365,7 @@
     group   => $config_file_group,
     mode    => $config_file_mode,
     content => template('ntp/ntp.conf.erb'),
-    require => Package[$package_name_real],
+    require => $ntp_conf_require,
```

Comment:
@ghoneycutt that's a neat idea :)
@ofsole I can help you with that if you want.